utopia-remix/server.js

```diff
@@ -180,8 +180,8 @@ app.use('/authenticate', proxy)
 app.use('/editor', proxy)
 app.use('/hashed-assets.json', proxy)
 app.use('/logout', proxy)
-app.use('/p', proxy)
-app.use('/project', proxy)
+app.use('/p/?$', proxy)
```

Comment:
this will leave only the empty `/p` links to go through our regular proxy (and not remix)

Reply:
It's worth adding a comment to that effect here